src/components/templates/Textarea/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from '@emotion/styled'
+
+const textArea = ({style, className, onChange, value, name , placeholder})=> {
```

Comment:
```suggestion
const TextArea = ({style, className, onChange, value, name , placeholder})=> {
```
컴포넌트명은 대문자로 해주시는게 좋을 것 같습니다! 😊